public/css/style.css

```diff
@@ -30,6 +30,17 @@ body {
   background-color: var(--white);
 }
 
+button#todaysDeck{
+  width: 150px;
+}
+
+button#login {
+  width: 140px;
+}
+
+button#logout {
+  width: 140px;
+
```

Comment:
I removed the explicit button width in one my previous PRs to keep the styling for the buttons consistent throughout. If this is a design choice please ignore me. :D

Reply:
I did not really want to have to add the explicit widths, however, it was the only way that I could get the text in the login button to be centred. Then to match so the login button was not wider than the logout button I changed that one as well just to keep it consistent. For the todays deck button, I added that because as it was we had a square looking button and I thought that widening it out slightly improved its look to be more consistent with out other buttons which are longer than they are tall.

Before:
![Screenshot 2023-05-02 at 4 46 42 PM](https://user-images.githubusercontent.com/120068344/235770060-283dcb49-111d-430b-99ee-0a74dc5e7468.png)

After:
![Screenshot 2023-05-02 at 4 46 49 PM](https://user-images.githubusercontent.com/120068344/235770132-1bcaf66b-46fc-406f-b34d-372da2a95216.png)


If you would prefer to leave it as-is, however, that is totally fine too!